website/docs/events/m3.md

```diff
@@ -0,0 +1,43 @@
+# Workshop: Multiplier Event 3
+
+## Schedule
+
+- Date: Thursday, March 2, 2023
+- Time: 10:00-12:00 EEST
+- Place: Online, on Google Meet
```

Comment:
Why not Discord?

Reply:
Many people don't have Discord or may find it difficult to find the right channel, etc. On the other hand, it may be a good PR move for us to bring more people onto our server. What do the others say?

Reply:
That's why I suggest Discord, for getting more people on the server.

Reply:
OK, I'm fine either way. I'll change it if the others agree with you.

Reply:
I agree with Discord for the same reason.

Reply:
I've changed it to Discord.